### PR TITLE
Fixing line breaks in invite email success page (chrome and safari)

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Fixing line breaks in invite email success page (chrome and safari)

#### Ticket Link
[MM-17193](https://mattermost.atlassian.net/browse/MM-17193)

### CODE REVIEW

File: components/invitation_modal/invitation_modal_confirm_step_row.scss [hunk 1]
```diff
@@ -9,6 +9,7 @@
         .email {
             margin: 0 4px 0 8px;
             overflow-wrap: anywhere;
```
Comment (jespino) [inline components/invitation_modal/invitation_modal_confirm_step_row.scss]: this line is for firefox

File: components/invitation_modal/invitation_modal_confirm_step_row.scss [hunk 2]
```diff
@@ -9,6 +9,7 @@
...
+            word-break: break-word;
```
Comment (jespino) [inline components/invitation_modal/invitation_modal_confirm_step_row.scss]: this line is for webkit (chrome and safari)

**Comment by prapti** [comment]
Long email addresses are getting wrapped as expected. I'll approve once all devs complete review.

File: components/invitation_modal/invitation_modal_confirm_step_row.scss (hunk 2 above)
Comment (lieut-data) [reply components/invitation_modal/invitation_modal_confirm_step_row.scss]: Almost a palindrome :D

**Pull request merged**